Makefile.am

```diff
@@ -2598,7 +2598,7 @@ ganeti:
 .PHONY: check-dirs
 check-dirs: $(GENERATED_FILES)
 	@set -e; \
-	find . -type d \( -name . -o -name .git -prune -o -print \) | { \
+	find . -type d -not -name '.' -not -path '*/\.*' | { \
```

Comment:
Would it be sufficient to limit this to the root directory? i.e.
`find . -type d -not -name '.' -not -path './.*`
would be a more conservative approach. ATM I can not image what tool might clutter subdirs with .dot-folders? Just for curiosity: must `.` be escaped in `-path`?

Reply:
Thanks for the hin, that was probably a bit too much. `find . -type d -not -name '.' -not -path './.*'` works just as good and is easier to read :-) I will fix that!

Reply:
While the `check-local` target is also intended to be run locally by the developer and therefore it sounds good to exclude all `.dot-folders`, here it will no longer check for `.hpc` directory, which is enumerate in: https://github.com/ganeti/ganeti/blob/72fcdea9139c22201d6b19cc60a0525c2a345fe4/Makefile.am#L270 To get `check-local` into the CI, I suggest a minimal invasive way by adding .github only to the excludes dirs:
```make
check-dirs: $(GENERATED_FILES)
        @set -e; \
        find . -type d \( -name . -o -name .git -o - name .github -prune -o -print \) | { \
``` 
Probably we can see later, how to exclude more things locally.

Reply:
Good catch, I'll update that accordingly!

Reply:
This has been fixed with the latest commit.